src/mgr/DaemonHealthMetricCollector.cc

```diff
@@ -77,12 +80,21 @@ class PendingPGs final : public DaemonHealthMetricCollector {
     if (osds.empty()) {
       return;
     }
-    check.summary = fmt::format("{} PGs pending on creation", value.n);
+    auto is_max_pgs_per_osd = value.n2;
+    check.summary =
+        fmt::format("{} PGs pending on creation {}",
+           value.n1,
+           (is_max_pgs_per_osd ? "(PG creation pending, " \
```

Comment:
You can probably create a string with `"(PG creation pending, temporarily increase 'osd_max_pg_per_osd_hard_ratio')"` and use it where appropriate.
This would help fix the indentations as well.